joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -253,6 +375,38 @@ JointTrajectoryController::on_configure(const rclcpp_lifecycle::State & previous
 
   last_state_publish_time_ = lifecycle_node_->now();
 
+  // action server configuration
+  {
+    if (lifecycle_node_->has_parameter("allow_partial_joints_goal")) {
+      allow_partial_joints_goal_ = lifecycle_node_->get_parameter("allow_partial_joints_goal")
+        .get_value<bool>();
+    }
+    if (allow_partial_joints_goal_) {
+      // TODO(ddengster): implement partial joints, log an enabled partial joints goal message
```

Comment:
please open a follow-up issue and link it here

Reply:
https://github.com/ros-controls/ros2_controllers/issues/30

Reply:
https://github.com/ros-controls/ros2_controllers/issues/31